_topics/en/threshold-signature.md

```diff
@@ -0,0 +1,126 @@
+## "[title](link)"
+#primary_sources:
+#    - title: Test
+#    - title: Example
+#      link: https://example.com
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: "Talk summary: The quest for practical threshold Schnorr signatures"
+    url: /en/newsletters/2019/10/16/#the-quest-for-practical-threshold-schnorr-signatures
+
+  - title: Paper published about ECDSA threshold signatures
+    url: /en/newsletters/2018/10/23/#two-papers-published-on-fast-multiparty-ecdsa
+
+  - title: Pairing-based threshold signatures with signer auditability
+    url: /en/newsletters/2018/10/09/#compact-multi-signatures-for-smaller-blockchains
+
+  - title: Discussion about digital signatures schemes, including threshold signatures
+    url: /en/newsletters/2020/07/01/#schnorr-signatures-and-multisignatures
+
+  - title: "Talk summary: Secure Protocols on bip-taproot / threshold sigs with MuSig"
+    url: /en/newsletters/2019/06/19/#secure-protocols-on-bip-taproot
+
+  - title: Cryptographic Vulnerabilities in Threshold Wallets
+    url: /en/newsletters/2019/06/19/#cryptographic-vulnerabilities-in-threshold-wallets
+
+## Optional.  Same format as "primary_sources" above
+see_also:
+  - title: Multisignature
+    link: topic multisignature
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Threshold signatures** are digital signatures that may have been
+  created by a subset of the private keys that could have created an
+  equivalent multisignature.  Like a multisignature, a threshold
+  signature can be verified using only a single public key and a single
+  signature.
+
+---
+Different algorithms exist for creating threshold signatures, but
+perhaps the simplest of these is a slight extension of a typical
+algorithm for creating a [multisignature][topic multisignature].  This
+is easiest explained with an example: three participants want to receive
+funds that can be spent by any three of them.  They cooperate to create
+an ordinary multisignature public key for receiving the funds, then they
+each take the extra step of deriving two [secret shares][vss] from their
+private key---one share for each of the other two participants.  The
+shares are created so that any two shares can reconstitute the private
+key that created it.  Each participant gives a different one of their
+secret shares to each other participant, so each participant ends up
+having to store their own private key plus one share for each other
+participant.  Each participant is then able to verify that the shares
+they received were correctly derived (i.e. not fake) and are unique from
+the shares received by the other participants.
+
+Later, participants A and B decide they want to create a signature
+even though the participant C isn't participating.  A and B
```

Comment:
Perhaps remove the repeated 'participants' here:

```suggestion
Later, A and B decide they want to create a signature
even though C isn't participating.  A and B
```

(would require introducing A, B and C further up)